### PR TITLE
Add types for CDN script of Telegram Web Bots called telegram-web-app.js

### PR DESCRIPTION
**Pull request by KnorpelSenf**
This PR adds type definitions for the CDN script used in Telegram Web Bots. It can be found at https://telegram.org/js/telegram-web-app.js. The documentation for it is available at https://core.telegram.org/bots/webapps.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/telegram-web-app/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,306 @@
+// Type definitions for non-npm package telegram-web-app 1.0
+// Project: https://telegram.org/js/telegram-web-app.js
+// Definitions by: KnorpelSenf <https://github.com/KnorpelSenf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+declare global {
+  interface Window {
+    Telegram: Telegram;
+  }
+}
```
Comment (peterblazejewicz) [inline types/telegram-web-app/index.d.ts:9]:
```suggestion

declare var Telegram: Telegram;
```
and remove every `export ` keyword. This should support both global script usage and usage via `window.Telegram` (`export` is not needed, as this is global script, not a module)

Comment (peterblazejewicz) [inline types/telegram-web-app/index.d.ts:9]:
```suggestion

declare var Telegram: Telegram;
```
and remove every `export ` keyword. This should support both global script usage and usage via `window.Telegram` (`export` is not needed, as this is global script, not a module)

Comment (KnorpelSenf) [reply types/telegram-web-app/index.d.ts:9]: That's what I tried, but it made the linting fail. Something like "export isn't needed, not you should add it for clarity".

Comment (KnorpelSenf) [reply types/telegram-web-app/index.d.ts:9]: Ah you mean declare instead of export? I'll have another look

**Review by laosb — APPROVED** [review]
LGTM

**Comment by peterblazejewicz** [comment]
please run:
```bash
npm run prettier -- --write ./types/telegram-web-app/**
```
https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes (para about formatting)
commit and push changes and it's ready to go

**Comment by KnorpelSenf** [comment]
@peterblazejewicz sorry for the delay, it should be ready to merge now.

File: types/telegram-web-app/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,290 @@
+// Type definitions for non-npm package telegram-web-app 1.0
+// Project: https://telegram.org/js/telegram-web-app.js
+// Definitions by: KnorpelSenf <https://github.com/KnorpelSenf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+declare var Telegram: Telegram;
```
Comment (peterblazejewicz) [inline types/telegram-web-app/index.d.ts:5]:
```suggestion

declare var Telegram: Telegram;
```
(minor stylistic update)

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@KnorpelSenf thanks!

**Comment by KnorpelSenf** [comment]
Ready to merge

**Pull request merged**